src/components/ItemEditorPage/LeftPanel/Items/Items.tsx

```diff
@@ -97,20 +129,46 @@ export default class Items extends React.PureComponent<Props, State> {
   }
 
   renderSidebarItem = (item: Item) => {
-    const { selectedCollectionId, selectedItemId, bodyShape } = this.props
+    const { selectedCollectionId, selectedItemId, bodyShape, isPlayingEmote } = this.props
     return (
       <SidebarItem
         key={item.id}
         item={item}
         isSelected={selectedItemId === item.id}
         isVisible={this.isVisible(item)}
+        isPlayingEmote={isPlayingEmote}
```

Comment:
I think I've found an issue with this logic: 
![image](https://user-images.githubusercontent.com/8763687/183071907-64764b33-5c9d-4676-8983-214bc6c1380b.png)
While trying different emotes, I noticed they are "sharing" the status, they all are paused/play at the same time (I've played each once).

I think it might be with this line, maybe we have to do something like this?

```
isPlayingEmote={isPlayingEmote && visibleItems.some(visibleItem => visibleItem.id === item.id ) }
```
And we should adpat the `visibleItems` to have just 1 emote at once.

Reply:
@juanmahidalgo good catch! Adapting the `visibleItems` to have just one emote at once, it is not required to add `visibleItems.some(visibleItem => visibleItem.id === item.id ) }`.